.github/workflows/python.yml

```diff
@@ -5,24 +5,43 @@ name: Python
 
 on: [push, pull_request]
 
+env:
+  PYTHON_MIN: '3.7'
```

Comment:
i added a pyproject.toml file and moved tool versioning info to that, so i think we can drop this now